tests/Integration/Behaviour/Features/Context/Domain/ProductFeatureContext.php

```diff
@@ -55,6 +58,53 @@ public function setProductTaxRulesGroup(string $taxRulesGroupReference, string $
         Cache::clean('product_id_tax_rules_group_*');
     }
 
+    /**
+     * @Then The category of product :productName has reduction of :reductionPercent% for the customer :customerReference
+     */
+    public function theCategoryOfProductHasReductionOfForCustomer(string $productName, float $reductionPercent, string $customerReference)
```

Comment:
The function name looks a bit strange, I can't find what it really does by reading it, and it looks like it's not really doing what the description/function name say

Reply:
renamed ;)